app/controllers/brexit_checker_controller.rb

```diff
@@ -52,12 +62,56 @@ def confirm_email_signup
 
   def save_results; end
 
+  def saved_results
+    redirect_to transition_checker_new_session_path(redirect_path: transition_checker_saved_results_path) and return unless logged_in?
+
+    if results_from_account.empty?
+      redirect_to transition_checker_questions_path
+    else
+      redirect_to transition_checker_results_path(c: results_from_account)
+    end
+  end
+
+  def edit_saved_results
+    redirect_to transition_checker_new_session_path(redirect_path: transition_checker_edit_saved_results_path) and return unless logged_in?
+
+    if results_from_account.empty?
+      redirect_to transition_checker_questions_path
+    else
+      redirect_to transition_checker_questions_path(c: results_from_account, page: 0)
+    end
+  end
+
 private
 
+  def results_from_account
+    @results_from_account ||= begin
+      Array(
+        update_session_tokens(
+          oidc.get_checker_attribute(
+            access_token: session[:access_token],
+            refresh_token: session[:refresh_token],
+          ),
+        ),
+      )
+                              rescue OidcClient::OAuthFailure
```

Comment:
This is some weird indentation

Reply:
Yikes, looks like `rubocop -a` did this in an attempt to keep the `rescue` in alignment with `begin`.